docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
 
     # chkconfig mysqld on
 
-Create Database
+Create database
 
     # mysqladmin -uroot create vnet
     # cd /opt/axsh/openvnet/vnet
     # bundle exec rake db:init
 
+To insert the database records use `vnctl`
+
+```
+# cd /opt/axsh/openvnet/vnctl
+# bundle install --path vendor/bundle
+```
 
-Start the OpenVNet services
----------------------------
+Start vnmgr and webapi since `vnctl` is just a webapi client.
 
     # initctl start vnet-vnmgr
     # initctl start vnet-webapi
-    # initctl start vnet-vna
-
-OpenVNet writes its logs in the /var/log/openvnet directory. If there's a problem starting any of the services, you can find its log files there.
-
-
-
----------------------------------------------------------------------
-
-# Let's try 1Box OpenVNet#
 
-This section explains how to install OpenVNet on one node. This is a great way to try it out.
-![1BOX](OpenVNet_1Box_environment.gif)
+From now on you need to create a couple of database records to launch your OpenVNet environment.
 
-Install the following operation system and kernel.
+#### Datapath
 
-+ RHEL (or Centos) 6.4
-+ Kernel 2.6.32-358.6.2.el6.x86_64
+```
+# ./bin/vnctl datapaths add --uuid dp-test1 --display-name test1 --dpid 0x0000aaaaaaaaaaaa --node-id vna
+```
 
+* dpid
 
-Installation and Setup of OpenVNet
------------------------------------
+The datapath ID specified in `/etc/sysconfig/network-scripts/ifcfg-br0`
 
-### Network interfaces setup
+* node-id
 
-We're going to enslave eth0 to openvswitch so OpenVNet can use it. You will no longer be able to use eth0 to ssh into the node or access the Internet. Prepare another interface for that if you need to.
+The ID of the vna written in `/etc/openvnet/vna.conf`
```

Comment:
We are describing node-id and dpid under the command. It's very likely that the reader has already copy pasted and run the command by now.